### PR TITLE
Assignment2- html

### PR DESCRIPTION
**Pull request by srigowrideo**

### CODE REVIEW

File: Srigowri_Deogirikar/html4.html [hunk 1]
```diff
@@ -16,6 +16,6 @@ <h4>Notepad or TextEdit </h4>
              <br>
              However, for learning HTML we recommend a simple text editor like Notepad(PC)or TextEdit(Mac)
             </p2>
```
Comment (Pawan1005) [inline Srigowri_Deogirikar/html4.html:18]: what is p2 tag

File: Srigowri_Deogirikar/HTMLAssignment2.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+At that time Larry Page,Sergey Brin, and Eric Schmidt agreed to work together at Google for <del>30</del>  <u>20</u>
+<br> years,until the year 2024.
+
+ </s2></p4>
+
+<h3><span style="color:darkolivegreen;">Thank You!!<span></h3>
+
+
+</body>
+</html>
```
Comment (Pawan1005) [inline Srigowri_Deogirikar/HTMLAssignment2.html:55]: add new line

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

File: Srigowri_Deogirikar/html4.html (hunk 1 above)
Comment (srigowrideo) [reply Srigowri_Deogirikar/html4.html:18]: Just for avoiding confusion with paragraph , i named it like that sir

**Review by srigowrideo — COMMENTED** [review]
sir, i changed it .

**Pull request merged**